client/securedrop_client/sdk/progress.py

```diff
@@ -0,0 +1,19 @@
+from PyQt5.QtCore import pyqtBoundSignal
+
+
+class ProgressProxy:
+    """
+    Relay the current download progress over to the UI; see
+    the FileDownloadProgressBar widget.
+    """
+
+    def __init__(self, inner: pyqtBoundSignal | None) -> None:
+        self.inner = inner
+
+    def set_value(self, value: int) -> None:
+        if self.inner:
+            self.inner.emit({"size": value})
+
+    def set_decypting(self) -> None:
```

Comment:
(sorry, smallest of nits: method name typo)

Reply:
oooops good catch